advntr/advntr_commands.py

```diff
@@ -88,7 +88,10 @@ def genotype(args, genotype_parser):
 
     log_file = working_directory + 'log_%s.log' % os.path.basename(input_file)
     log_format = '%(asctime)s %(levelname)s:%(message)s'
-    logging.basicConfig(format=log_format, filename=log_file, level=logging.DEBUG, filemode='w')
+    level = logging.DEBUG
+    if args.disable_logging:
+        level = logging.CRITICAL
+    logging.basicConfig(format=log_format, filename=log_file, level=level, filemode='w')
```

Comment:
Can we use a different method, for example [logging.disable()](https://docs.python.org/3/library/logging.html#logging.disable)?

As we haven't logged any CRITICAL-level log, this will solve the issue, but it seems like there is a function exactly what we want.

Reply:
The logic for this was to have one source of logging even when we disable the majority of the logs for very critical logs. But I agree that it's confusing if the flag is not fully disabling the logs. So I changed it now to disable completely.